src/ws/session.c

```diff
@@ -78,17 +82,19 @@ read_auth_until_eof (size_t *out_len)
       if (alloc <= len)
         {
           alloc += 1024;
+          if (alloc > 64 * 1024)
+            errx (EX, "input data is too long %s", what);
```

Comment:
style of message inconsistent with allocate error: ... `too long for %s` would be consistent